filesystem/config.pan

```diff
@@ -151,13 +151,29 @@ variable DISK_SWAP_SIZE ?= {
   swap_size;
 };
 
+
+# Variables related to GPT biosboot/UEFI support
+
+@{
+desc = indicates if a UEFI boot is used
+values = boolean
+default = false
+required = no
+}
+variable DISK_BIOS_TYPE_UEFI ?= false;
+
 @{
 desc =  default size for block device biosboot if created
 values = long
-default = 100 MB
+default = 100 MB for legacy BIOS, 200 MB for UEFI
 required = no
 }
-variable DISK_BIOSBOOT_BLOCKDEV_SIZE_DEFAULT ?= 100*MB;
+variable DISK_BIOSBOOT_BLOCKDEV_SIZE_DEFAULT ?= if ( DISK_BIOS_TYPE_UEFI ) {
+                                       200*MB;
+                                     } else {
+                                       100*MB;
```

Comment:
This is what I already heard several times but Adrien who gave me his suggestions told me 100 MB... I'll check with him and update it.